src/sass/base/_va.scss

```diff
@@ -21,6 +22,10 @@ body {
 }
 
 // Skip link
+.show-on-focus-wrapper {
+  padding: 1em;
+}
```

Comment:
Looks like this padding is showing when at all times, instead of just when focused.

Reply:
Good catch @ncksllvn, fixed.